yoti-stub/src/data/getSessions/aiFailManualFail.ts

```diff
@@ -0,0 +1,280 @@
+export const AI_FAIL_MANUAL_FAIL = {
```

Comment:
are these new cases being used? 🤔

Reply:
Not currently but they've been provided to us by Yoti so I thought I'd throw them in now to be used potentially in near future, happy to remove them though